traffic_monitor/config/config.go

```diff
@@ -177,6 +180,9 @@ func (c *Config) UnmarshalJSON(data []byte) error {
 	if aux.PeerOptimistic != nil {
 		c.PeerOptimistic = *aux.PeerOptimistic
 	}
+	if aux.TrafficOpsRetryIntervalSec != nil {
```

Comment:
I think we should also protect against 0 or negative values here which would cause time.Sleep() to return immediately and hammer TO repeatedly. So maybe we should set it to the default of 3 seconds in that case and log a warning that the value needs to be positive.

Reply:
good catch, i'll make the change